README.md

```diff
@@ -12,6 +12,18 @@ ESLint is a tool for identifying and reporting on patterns found in ECMAScript/J
 2. Run `codeclimate engines:enable eslint`. This command both installs the engine and enables it in your `.codeclimate.yml` file.
 3. You're ready to analyze! Browse into your project's folder and run `codeclimate analyze`.
 
+### Configuration
+
+You can configure our ESLint Engine to analyze the file types you’d like by configuring the extensions in your `.codeclimate.yml`, as shown in the example below:
+
+```eslint:
```

Comment:
Formatting on this block is off: first line of content should start on a new line after the <code>```</code>, sub-values should be indented 2 spaces instead of 1, and the extension lines starting with `-` should line up with `extensions`. Like:

<pre>
```yaml
eslint:
  enabled: true
  config:
    extensions:
    - .es6
    - .js
    - .jsx
```
</pre>